### PR TITLE
feat(pacstall): Add argument parser

### PR DESCRIPTION
**Pull request by wizard-28**
## Purpose

Adds argument parser to `pacstall.py`

## Approach

Using the `argparse` standard lib.

## Checklist

- [x] I confirm that I have read the [contributing guidelines](https://github.com/pacstall/pacstall/blob/develop/CONTRIBUTING.md), and this pull request is abiding by all the clauses stated in the guideline.

### CODE REVIEW

**Comment by Henryws** [comment]
Fix codefactor issues

File: src/pacstall.py [hunk 1]
```diff
@@ -41,4 +65,59 @@
 ]:
     message.fancy("info", "APT lists were updated more than a week ago")
     message.fancy("info", "Updating APT lists")
-    os.system("sudo apt-get -qq update")
+    os.system("/usr/bin/sudo /usr/bin/apt-get -qq update")
```
Comment (Henryws) [inline src/pacstall.py:68]: Why hardcoded paths? Does it not work without?

Comment (Henryws) [reply src/pacstall.py:68]: How can you be sure that `/usr/bin/sudó` exists?

**Review by D-Brox — APPROVED** [review]

File: src/pacstall.py (hunk 1 above)
Comment (wizard-28) [reply src/pacstall.py:68]: Because it does? Duh.

**Review by Henryws — APPROVED** [review]

**Pull request merged**